src/test/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfigurationTest.java

```diff
@@ -0,0 +1,42 @@
+package org.datadog.jenkins.plugins.datadog;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import hudson.util.Secret;
+import org.jenkinsci.plugins.plaincredentials.impl.StringCredentialsImpl;
+import org.jenkinsci.plugins.plaincredentials.StringCredentials;
+
+import com.cloudbees.plugins.credentials.CredentialsProvider;
+import com.cloudbees.plugins.credentials.CredentialsScope;
+import com.cloudbees.plugins.credentials.CredentialsStore;
+import com.cloudbees.plugins.credentials.domains.Domain;
+
+import org.junit.Assert;
+import java.io.IOException;
+
+public class DatadogGlobalConfigurationTest {
+
+    @ClassRule
+    public static JenkinsRule jenkinsRule = new JenkinsRule();
+
+    @Test
+    public void testCanGetCredentialFromId() throws IOException {
+        CredentialsStore credentialsStore = CredentialsProvider.lookupStores(jenkinsRule).iterator().next();
+
+        StringCredentials credential1 = new StringCredentialsImpl(CredentialsScope.SYSTEM, "string-cred-id", "description", Secret.fromString("api-key"));
+        credentialsStore.addCredentials(Domain.global(), credential1);
+        DatadogGlobalConfiguration cfg = DatadogUtilities.getDatadogGlobalDescriptor();
+        Assert.assertTrue(cfg.getCredentialFromId("string-cred-id").equals(credential1));
```

Comment:
Could we have a test to check what happens if the `credential-id` does not exist?

Reply:
Added!